speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/FileDataStorage.h

```diff
@@ -15,6 +19,31 @@ class FileDataStorage
     tDataToStore m_saveSettings;
     /// @brief Output filestream to write data to, should be initialized through @link FileDataStorage::Initialize
     std::ofstream m_outputStream = {};
+    /// @brief Integer that determines the compression rate
+    int m_compressionRate = 1;
+    /// @brief Integer that determines the current step of the compression
+    unsigned long m_compressionStep;
```

Comment:
It is better to keep this as an int, it prevents a lot of casting in other files and it should not matter as the maximum compression is ~50